src/index.html

```diff
@@ -7,16 +7,449 @@
+              tracking so you know where your bike is and can track it anytime.
+            </p>
+          </div>
+        </article>
+
+        <article class="article">
+          <div class="figure article__figure">
+            <div class="photo-wrapper article__figure--big">
+              <img
+                class="detail-img"
+                alt="product-bike"
+                src="./images/elThree.jpg"
+              />
+            </div>
+            <div class="photo-wrapper article__figure--small">
+              <img
+                class="detail-img"
+                alt="product-bike"
+                src="./images/elFour.jpg"
+              />
+            </div>
+          </div>
+          <div class="article__description">
+            <h4 class="title title--fourth article__title">
+              Range & Integrated lights
+            </h4>
+            <p
+              class="content-paragraph content-paragraph--description article__paragraph"
+            >
+              The removable battery has up to 70km battery autonomy and weighs
+              only 2.4 kg. Lights integrated into the frame give you always-on
+              visibility day and night.
+            </p>
+          </div>
+        </article>
+
+        <article class="article">
+          <div class="figure article__figure">
+            <div class="photo-wrapper article__figure--small">
+              <img
+                class="detail-img"
+                alt="product-bike"
+                src="./images/elFive.jpg"
+              />
+            </div>
+            <div class="photo-wrapper article__figure--big">
+              <img
+                class="detail-img"
+                alt="product-bike"
+                src="./images/elSix.jpg"
+              />
+            </div>
+          </div>
+          <div class="article__description">
+            <h4 class="title title--fourth article__title">
+              Hydraulic disc brakes & Lightweight
+            </h4>
+            <p
+              class="content-paragraph content-paragraph--description article__paragraph"
+            >
+              Brakes with total stopping power the second you make contact. The
+              removable battery has up to 70km battery autonomy and weighs only
+              2.4 kg. Lights integrated into the frame give you always-on
+              visibility day and night.
+            </p>
+          </div>
+        </article>
+      </section>
+      <button
+        class="button content__button"
+        type="button"
+      >
+        Explore
+      </button>
+    </main>
+
+    <section
+      class="content__section content__section--template content__contact"
+      id="contact-us"
+    >
+      <h3 class="title title--third contact-title title__ipad">Contact us</h3>
+      <form
+        action="#"
+        onsubmit="event.preventDefault()"
+        class="form"
+      >
```

Comment:
to reset form after submit you can add this to main.js file:

```
const form = document.querySelector('form');
form.addEventListener('submit', (event) => {
  event.preventDefault();

  form.reset();
});
```

```suggestion
      <form
        action="#"
        class="form"
      >
```